### PR TITLE
Fix partner images and 404

### PR DESCRIPTION
**Pull request by guzman-raphael**
- Serve in subdirectory to fix 404 issue
- Fix centered images in figures
- Add HubSpot chatbot

> :warning: Note: Upon merging, we should update our [ingress](https://github.com/datajoint-company/dj-gitops/blob/0e3378e1d05dc0ba828a6180c2176e9f6b66dff0/applications/k8s/deployments/documentation/ingress.yaml#L35) to reflect the new directory scheme from MkDocs Material
> from:
> ```nginx
> rewrite ^/[^/]+(/.*)$ /datajoint-docs$1 break;
> ```
> to:
> ```nginx
> rewrite ^/[^/]+(/.*)$ /datajoint-docs/docs$1 break;
> ```

Depends on datajoint-company/dj-gitops#60

### CODE REVIEW

**Review by dimitri-yatsenko — DISMISSED** [review]

File: mkdocs.yaml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # ---------------------- PROJECT SPECIFIC ---------------------------
 
 site_name: DataJoint Documentation
-repo_url: https://github.com/datajoint-company/datajoint-docs
+site_url: http://localhost/docs
```
Comment (kabilar) [inline mkdocs.yaml:4]:
```suggestion
site_url: https://www.datajoint.com/docs
```

Comment (kabilar) [reply mkdocs.yaml:4]: Perhaps this suggestion is not required based on how we deploy the site.

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @guzman-raphael!  One minor comment.

File: mkdocs.yaml (hunk 1 above)
Comment (guzman-raphael) [reply mkdocs.yaml:4]: Thanks @kabilar but this was intentional. Using `localhost` should protect us from future updates to the domain without requiring cascading PR's (though subdomain is a different story). I don't expect it to affect the deployment.

Comment (kabilar) [reply mkdocs.yaml:4]: Thank you

**Review by kabilar — APPROVED** [review]

**Pull request merged**